docs/docs/policies.mdx

```diff
@@ -325,6 +325,15 @@ a fixed behavior. It makes predictions based on any `rules` you have in your
 training data. See the [Rules documentation](./rules.mdx) for further information
 on how to define rules.
 
+You can remove rule restriction and check for contradictions by setting the following
```

Comment:
Can this be explained in two separate sub-sections? That way there can be a full definition of what each param does (e.g. what is the rule restriction?) and why you would use it.